common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3113,8 +3113,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_COMPACTOR_COMPACT_MM("hive.compactor.compact.insert.only", true,
         "Whether the compactor should compact insert-only tables. A safety switch."),
     COMPACTOR_CRUD_QUERY_BASED("hive.compactor.crud.query.based", false,
-        "Means Major compaction on full CRUD tables is done as a query, "
-        + "and minor compaction will be disabled."),
+        "Means compaction on full CRUD tables is done as a query. "
+        + "Compactions on insert-only tables will always run as a query."),
```

Comment:
Thanks for taking care of this!

I'd add to the end: "regardless of the value of this configuration" just so it's clear.

and "as **a** query" isn't exactly accurate, I'd change it to something like: "via queries"